WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/CoinJoinHistoryItemViewModel.cs

```diff
@@ -29,14 +29,7 @@ public class CoinJoinHistoryItemViewModel : HistoryItemViewModelBase
 		ConfirmedToolTip = $"{confirmations} confirmation{TextHelpers.AddSIfPlural(confirmations)}";
 
 		var amount = transactionSummary.Amount;
-		if (amount < Money.Zero)
-		{
-			OutgoingAmount = amount * -1;
-		}
-		else
-		{
-			IncomingAmount = amount;
-		}
+		SetAmount(amount);
```

Comment:
![image](https://user-images.githubusercontent.com/16364053/203937049-3b6253d2-ce26-413b-8148-1d3088922822.png)

Can you make the method not virtual? I don't think that is necessary.

Reply:
Done in https://github.com/zkSNACKs/WalletWasabi/pull/9618/commits/bda57c19d06c5bc2aab11397a2622eeca5071625